resources/views/vendor/backpack/theme-tabler/inc/sidebar_content.blade.php

```diff
@@ -1,118 +1,156 @@
 <li class="nav-item">
     <a class="nav-link" href="{{ backpack_url('dashboard') }}">
-        <i class="nav-icon fs-2 me-2 la la-dashboard"></i> <span>{{ trans('backpack::base.dashboard') }}</span>
+        <i class="nav-icon la la-dashboard"></i> <span>{{ trans('backpack::base.dashboard') }}</span>
     </a>
 </li>
 
 @includeWhen(class_exists(\Backpack\DevTools\DevToolsServiceProvider::class), 'backpack.devtools::buttons.sidebar_item')
 
-<li class="nav-separator">First-Party Addons</li>
+@if(!\Backpack\ThemeTabler\ThemeOptions::isHorizontalLayout())
```

Comment:
@maurohmartinez we can't do this in the `sidebar_content.blade.php`. If we use `\Backpack\ThemeTabler\ThemeOptions::class` here... we assume the Tabler theme exists... and we can't assume that, right?! Then we won't be able to switch between themes. So we have to do a `class_exists(\Backpack\ThemeTabler\ThemeOptions::class)` before this... but then it becomes too big... so how about we define a function at the start of this file? Could be something like:

```php
@php
function layout_is_horizontal() {
    if (!class_exists(\Backpack\ThemeTabler\ThemeOptions::class)) {
        return false;
    }

    return \Backpack\ThemeTabler\ThemeOptions::isHorizontalLayout();
}
@endphp
```

Reply:
Oh wait no. I'm stupid 😅 This file only gets loaded when `theme-tabler` is used 😅